src/pages/box-shadow.ts

```diff
@@ -69,13 +69,13 @@ export function boxShadowGenerator(
   const color = getShadowColor(attribute);
   const getOutputElement = getOutput(attribute);
   const resultPage = getResultPage();
-  
+
   const element = boxshadowInputs[0];
   const value = element.value;
 
   if (value.length < 3) {
     getOpenSideBarButton().style.display = 'none';
-  }else{
+  } else {
```

Comment:
here too

Reply:
Hey @Dun-sin. I tried to removed this but after commit it changing to this state again

Reply:
remove it from github itself click on the line and go `ctrl+g`

Reply:
worked , thanks